packages/synapse-react-client/test/containers/queryUtils.test.ts

```diff
@@ -309,6 +352,29 @@ describe('facet support', () => {
         true,
       )
     })
+    test('dataset collection cannot be added to download list', () => {
```

Comment:
May want to add a comment that points to SWC-6927

Reply:
```suggestion
    test('dataset collection cannot be added to download list (SWC-6927)', () => {
```